PotLock/activity-dao
# Tab routes

## Summary
Added guild logo and added tab queries.
interest tab now re-routes to interest/{id_slug}

## Related Issue
#44 
#56 

## Type of Change



## Testing Steps
npm run dev
check daos tab and check tabs from daos. 
guild logo now on bottom right that links to activityDao guild page.
